src/core/core.scale.js

```diff
@@ -594,7 +597,18 @@ module.exports = function(Chart) {
 						context.font = tickLabelFont;
 						context.textAlign = (isRotated) ? "right" : "center";
 						context.textBaseline = (isRotated) ? "middle" : options.position === "top" ? "bottom" : "top";
-						context.fillText(label, 0, 0);
+						
+						lineHeight = context.measureText("M").width * 1.2;
```

Comment:
good point. I its a hangover from the PoC but I can remove it now I'm calculating in the helpers